elasticdl/docker/Dockerfile.ci

```diff
@@ -31,7 +31,11 @@ RUN python -m pip install --quiet /elasticdl-develop-py3-none-any.whl \
     && rm /elasticdl-develop-py3-none-any.whl
 
 # Add elasticdl_ps to system path
-ENV PATH /usr/local/lib/python3.6/dist-packages/elasticdl/go/bin:$PATH
+RUN /bin/bash -c \
+        'PYTHON_PKG_PATH=$(pip3 show elasticdl | grep "Location:" | cut -d " " -f2); \
+        echo "PATH=${PYTHON_PKG_PATH}/elasticdl/go/bin:$PATH" >> \
+            /root/.bashrc_elasticdl; \
+        echo ". /root/.bashrc_elasticdl" >> /root/.bashrc'
```

Comment:
We are using `.bashrc_elasticdl` now, do we still need to append this PATH to `.bashrc`?

Reply:
In order to make sure that while running `docker run --rm -it <image_name> bash`, the `elasticd_ps` binary path will be appended to PATH environment variable. `source /root/.bashrc` is automatically executed when we runs the command above.